### ISSUE TITLE
OpenEXR library in PyIlmBase pkg-config

### ISSUE THREAD

**Issue by waebbl** [AcademySoftwareFoundation/openexr#1066]
Today I noticed there's a reference to `-lIlmImf` in the pkg-config file for PyIlmBase (v2.5). For a successful build PyIlmBase only needs to have IlmBase installed, but not OpenEXR, so I don't understand, why consumers should depend on OpenEXR as well. My question now, is this reference correct?

https://github.com/AcademySoftwareFoundation/openexr/blob/8bc3741131db146ad08a5b83af9e6e48f0e94a03/PyIlmBase/PyIlmBase.pc.in#L15

**Comment by cary-ilm** [comment]
That's true, no need for -lIlmImf, although that's the least of the problems here, since it's also missing the -lPyImath_Python* and -lPyIex libraries.  The PyIlmBase library primarily installs python modules and is almost never used as a C++ library, so I doubt the pkg-config is ever used. Which must be the case because it's rather hopelessly broken.

The PyImath pkg-config works in the 3.0 release of Imath, which I think should be an adequate fix, since that supersedes 2.5. The 2.5 setup attempts to support both CMake *and* autoconf, which was retired in the 3.0 release. Getting this to work across CMake and autoconf, as well as python2 and python3 is likely to be hard enough that I'd rather file this as "won't fix."

**Comment by cary-ilm** [comment]
Culling old issues. Closing this as the 2.x release is no longer supported. Feel free to re-open if you'd like further help.
